src/modules/historyEventTypes.ts

```diff
@@ -60,6 +60,7 @@ interface HistoryEventsData {
             buddyLogin?: string;
             coordinatorLogin?: string;
             recruiterLogin?: string;
+            supplementalPositions?: string;
```

Comment:
We can store array of objects here, no problem. Maybe we even extract analytics from them later